app/transport/http/resp.go

```diff
@@ -0,0 +1,25 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func GenerateJSONResponse(w http.ResponseWriter, statusCode int, data interface{}) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(statusCode)
+
+	if data == nil {
+		log.Println("Data can not be empty")
```

Comment:
why log? if we have 200 statuscode OK. Do we need to responce data?

Reply:
We always must to responce sommething. If data is empty we must know about it, thats why we use log now. Later we use some another logger